indico/web/templates/forms/link_list_widget.html

```diff
@@ -2,13 +2,13 @@
 
 {% block html %}
     <div id="{{ field.id }}-wrapper"></div>
+    <input type="hidden" id="{{ field.id }}" name="{{ field.id }}" value="{{ field.data | tojson | forceescape }}" />
```

Comment:
I don't remember the exact difference but usually we use `field._value()` instead of `field.data` here

Reply:
It seems that for `JSONField`s, `field._value()` returns the data in escaped json. Quite convenient, thanks!

Reply:
It's just dumped, you still need `|forceescape`